app/org/sagebionetworks/bridge/dynamodb/DynamoUploadDao.java

```diff
@@ -148,9 +148,7 @@ public Upload getUpload(@Nonnull String uploadId) {
 
         String nextPageOffsetKey = (page.getLastEvaluatedKey() != null) ? page.getLastEvaluatedKey().get(UPLOAD_ID).getS() : null;
 
-        int total = mapper.count(DynamoUpload2.class, createCountQuery(studyId.getIdentifier(), startTime, endTime));
-
-        return new PagedResourceList<>(uploadList, null, pageSize, total)
+        return new PagedResourceList<>(uploadList, null, pageSize, uploadList.size())
```

Comment:
If there's more records than are returned in one page, total will be wrong. Total is so that the client can correctly render the total number of pages that are present... in paged resource lists, it's not the same as the number of records in a page.